src/code_e92d0_len_5da0.c

```diff
@@ -673,13 +858,65 @@ ApiStatus si_handle_call(ScriptInstance* script) {
     return func(newScript, isInitialCall); // todo fake match
 }
 
-INCLUDE_ASM(s32, "code_e92d0_len_5da0", si_handle_exec1);
+ApiStatus si_handle_exec1(ScriptInstance* script) {
+    ScriptInstance* newScript;
+    s32 i;
+
+    newScript = start_script_in_group((ScriptInstance*)get_variable(script, *script->ptrReadPos), script->priority, 0,
+                                      script->groupFlags);
+
+    newScript->ownerActorID = script->ownerActorID;
+    newScript->ownerID = script->ownerID;
+
+    i = 0;
```

Comment:
the loops here are very weird... I assume they don't match as for loops? Seems kinda weird.

Reply:
yeah, I thought they didn't work normally but I guess I was wrong